### PR TITLE
Add Basic Canary Config

### PR DESCRIPTION
**Pull request by sethAmazon**
# Description of the issue
Canary has no metrics about the agent. 

# Description of changes
Add procstat metrics about the agent. 

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Manual run the canary test.

### CODE REVIEW

File: test/canary/resources/canary_config.json [hunk 1]
```diff
@@ -1,9 +1,28 @@
 {
   "metrics": {
```
Comment (jefchien) [inline test/canary/resources/canary_config.json:2]: Need to include aggregate dimensions so we can get a metric without dimensions. Looking at the canary metrics from its brief run, it adds a metric with a couple dimensions including the host IP, which makes it difficult to use.

File: test/canary/resources/canary_config.json [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
     "metrics_collected": {
-      "mem": {
-        "measurement": [
-          "mem_used_percent"
+      "procstat": [
+        {
+          "pid_file": "/opt/aws/amazon-cloudwatch-agent/var/amazon-cloudwatch-agent.pid",
+          "measurement": [
+            "cpu_usage",
+            "memory_data"
```
Comment (jefchien) [inline test/canary/resources/canary_config.json:10]: We also typically use `memory_rss`.

File: test/canary/resources/canary_config.json [hunk 3]
```diff
@@ -1,9 +1,28 @@
...
+          ]
+        }
+      ]
+    }
+  },
+  "logs": {
+    "logs_collected": {
+      "files": {
+        "collect_list": [
+          {
```
Comment (SaxyPandaBear) [inline test/canary/resources/canary_config.json:20]: nit - is there any reason we wouldn't want log group retention for this?

File: test/canary/resources/canary_config.json (hunk 1 above)
Comment (sethAmazon) [reply test/canary/resources/canary_config.json:2]: Added

**Review by jefchien — APPROVED** [review]

**Review by SaxyPandaBear — APPROVED** [review]
I don't think we need it for 14 days. Maybe 3 or less. But it's fine

**Pull request merged**